site/en/docs/multidevice/overview/index.md

```diff
@@ -10,29 +10,26 @@ Google provides three ways to run web content on tablets, smartphones, and other
 - Using Chrome Mobile for devices running Android 4.0 and later, and iOS.
```

Comment:
Should this file be also moved out of /overview/ dir?

Reply:
No, because the multidevice folder above it actually has a valid index page.